expat/lib/xmlparse.c

```diff
@@ -7124,7 +7124,7 @@ static unsigned long FASTCALL
 hash(XML_Parser parser, KEY s) {
   struct siphash state;
   struct sipkey key;
-  (void)sip24_valid;
+  (void)sip24_valid();
```

Comment:
@jblaisdev on a side note, this would now call the function while it was previously only marked as unused and not called. Given the place of this function call, if I had merged that change without noticing, this mistaken warning fix would have hurt Expat performance. Fixing warnings needs 110% attention to make things worse.